data/os/Debian/10.yaml

```diff
@@ -0,0 +1,2 @@
+---
+openldap::client::package: "libldap-2.4-2"
```

Comment:
Please fix line ending (better check your editor configuration to ensure it always does the right thing).  This file and a few others.

Trivia: POSIX definition of a [Line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) and an [Incomplete Line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_195) :-)

Reply:
Woops, never noticed VS Code was doing this before. Thx :)

Reply:
Yeah, some editors have bad defaults :disappointed:

Thanks for fixing it!